### ISSUE TITLE
Wallpaper Changing

### ISSUE THREAD

**Issue by syssgx** [mozilla/r2d2b2g#86]
You can right click on the home screen and choose a new wallpaper, only once.
When you do that, and then try to right click and choose another wallpaper, it won't respond, nor will the "wallpaper app" close through the button (upper-left).

**Comment by mykmelez** [comment]
I see that too! It looks like you can select another wallpaper by restarting the simulator first, so that's a workaround for the problem.

I also tried this on my B2G testing phone, and I see the same unresponsiveness the second time I try to pick a wallpaper; but after a few seconds the wallpaper picker app seems to crash and automatically restart, after which I'm able to pick another wallpaper.

This is the kind of B2G bug that doesn't seem like it'll affect app developers testing their apps in the simulator, so I'm inclined to say that we won't try to fix the problem, since our goal is to enable app developers to test their apps in Firefox OS, and it's a non-goal for us to enable users to test Firefox OS generally. But cc:ing @dangoor for his confirmation.

**Comment by syssgx** [comment]
good to know. yep, this project is just to bring b2g on the desktop.
